Release_Guideline.md

```diff
@@ -32,7 +32,7 @@ Patch releases will generally be more frequent, and indicate less significant ch
 The following is a non-exhaustive set of justifications for a patch release:
 
 - A modality field has changed and the [bids-validator](https://github.com/bids-standard/bids-validator) has been updated to reflect this change.
-- Links or information in the specification are no longer accurate, *e.g.* if a BEP document is added or moved
+- Links or information in the specification are no longer accurate, *for example* if a BEP document is added or moved
```

Comment:
Can the italics be dropped? I assume they were there before because "e.g." is Latin.

Reply:
:+1: I pretty much ignored the non-spec bits.

Reply:
They are a couple of italics in the non-specs part. 

I left them because they were there before indeed. :-)

Will remove them